src/Pos/Generator/Block/Mode.hs

```diff
@@ -317,7 +343,11 @@ instance MonadBlockGenBase m => MonadSlots (BlockGenMode m) where
     currentTimeSlotting = currentTimeSlottingSimple
 
 instance MonadBlockGenBase m => DB.MonadGState (BlockGenMode m) where
-    gsAdoptedBVData = gsAdoptedBVDataDefault
+    gsAdoptedBVData =
+        -- Currently we can't and don't want to test the system in boot mode
+        -- since in requires transaction to have specific distribution.
+        gsAdoptedBVDataDefault <&>
+        (\BlockVersionData{..} -> BlockVersionData { bvdUnlockStakeEpoch = 0, ..})
```

Comment:
1. It's very bad, because it differs from the data we store in DB and leads to inconsistent behavior and invariants violation.
2. Can we stop this cancer already? I can't stand it anymore. It's July 19 already. Why do we disable bootstrap era when something doesn't work and pretend it's a proper fix? When will we test bootstrap era? We must consider bootstrap era in our tests. It's even more important than past-bootstrap era.
Instead of writing proper code we add more and more hacks and pretend it's fine. We will not have time to rewrite it. We need to do it properly from the beginning.

> we can't

we can

> don't want

we want